internal/v1/handler_blueprints.go

```diff
@@ -94,6 +136,20 @@ func (h *Handlers) CreateBlueprint(ctx echo.Context) error {
 		desc = *blueprintRequest.Description
 	}
 
+	if blueprintRequest.Customizations.Users != nil {
```

Comment:
:shrug:  I am little scared to include more in this PR, but I am open to do it separately :D

Reply:
Yeah I guess that make sense, tho will we be able to have a follow-up this week?
I'm bit reluctant to introduce this to prod without filtering the password out.
We could also have a comment on the password field it's experimental :shrug:

Reply:
Resolving this, we decided to do it in this PR just separate commits, the part one adding the possibility of password and hashing it is done, I will do the 'filtering out' as separate commit, we talked and it should just return some constant instead of hashed password (with a note to take care of update accordingly)